ztp/policygenerator/policyGen/policyBuilder.go

```diff
@@ -94,6 +106,18 @@ func (pbuilder *PolicyBuilder) Build(policyGenTemp utils.PolicyGenTemplate) (map
 					if err != nil {
 						return policies, err
 					}
+					if sFile.RemediationAction != utils.UnsetStringValue {
+						_, exists := remediationActionMap[sFile.PolicyName]
+						if exists && sFile.RemediationAction != policies[output].(utils.AcmPolicy).Spec.RemediationAction {
```

Comment:
No need for the remediationActionMap. condition should be like that
```
if sFile.RemediationAction != acmPolicy.Spec.RemediationAction && acmPolicy.Spec.RemediationAction != policyGenTemp.Spec.RemediationAction {
// raise error.
}
```

Reply:
works! :)